src/schema/v2/partner/partner.ts

```diff
@@ -752,6 +753,41 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLString,
         resolve: ({ collecting_institution }) => collecting_institution,
       },
+      contactsConnection: {
+        description: "A connection of contacts from a Partner.",
+        type: contactsConnection.connectionType,
+        args: pageable({
+          page: {
+            type: GraphQLInt,
+          },
+          size: {
+            type: GraphQLInt,
+          },
+        }),
+        resolve: async ({ id }, args, { partnerContactsLoader }) => {
+          if (!partnerContactsLoader) return null
+
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          const { body } = await partnerContactsLoader(id, {
+            page,
```

Comment:
You likely need to also pass in `total_count: true` to get the count from the headers.